.github/workflows/build.yml

```diff
@@ -108,13 +107,12 @@ jobs:
       - name: Set-up Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: lts/*
+          node-version: latest
```

Comment:
@smorimoto, diff to the yml files have nothing to do with this PR.
Can you keep the scope of PRs to the minimum, please, explaining in the description the reason for the CHANGES.
For example, why should we remove the dune cache ? I tink I know the reason but it forces me to dig information too much.

Reply:
Sure